scripts/microbe_stage/organelle_components/movement_organelle.as

```diff
@@ -189,10 +190,10 @@ class MovementOrganelle : OrganelleComponent{
         auto renderNode = organelle.world.GetComponent_RenderNode(organelle.organelleEntity);
         if (renderNode !is null && IsInGraphicalMode())
         {
-            renderNode.Node.setPosition(membraneCoords);
-            renderNode.Node.setOrientation(bs::Quaternion(bs::Degree(180),
-                        bs::Vector3(0, 1, 0)) * bs::Quaternion(bs::Degree(angle),
-                            bs::Vector3(0, 0, 1)));
+        renderNode.Node.setPosition(membraneCoords);
```

Comment:
One level of indentation is gone from here too.

Reply:
fixed